bundle/manifests/sysdig-operator.clusterserviceversion.yaml

```diff
@@ -36,7 +36,7 @@ metadata:
     operators.operatorframework.io/project_layout: helm.sdk.operatorframework.io/v1
     repository: https://github.com/sysdiglabs/sysdig-operator
     support: Sysdig, Inc.
-  name: sysdig-operator.v1.12.40
+  name: sysdig-operator.v1.12.43
```

Comment:
Same comment as above, but this one seems harder, as I don't think we can run any command in the .yaml

Reply:
Another comment, isn't the submodule folder added to the repository as part of the commit to freeze the specific commit of the submodule? See an example in here: https://github.com/sysdiglabs/demoenv-partners/tree/master/apps/build/files

Otherwise I am afraid that you'll need to initialize the submodule everytime the repository is cloned, and it will checkout the master branch.

Reply:
> Another comment, isn't the submodule folder added to the repository as part of the commit to freeze the specific commit of the submodule? See an example in here: https://github.com/sysdiglabs/demoenv-partners/tree/master/apps/build/files
> 
> Otherwise I am afraid that you'll need to initialize the submodule everytime the repository is cloned, and it will checkout the master branch.

Dismiss this, the folder is there even if there is not shown as part of the changes in the PR diff viewer.

Reply:
Once the automated version bump is integrated from the above comment, we will be able to run the bundle build to update these versions. The only issue so far has been that sometimes bundle versions can diverge from chart versions for bundle-specific bug fixes, but those are not frequent